openshift/deployment-template.yaml

```diff
@@ -213,11 +235,4 @@ objects:
               - isis-model-download
             from:
               kind: ImageStreamTag
-              name: 'isis-api:${IMAGE_STREAM_TAG}'
-
-parameters:
-  - description: Tag of the ImageStream to use
-    displayName: ImageStream Tag
-    required: true
-    name: IMAGE_STREAM_TAG
-    value: 'latest'
+              name: "${IMAGE_STREAM_NAMESPACE}/isis-api:${IMAGE_STREAM_TAG}""
```

Comment:
@goern I m not sure if this will suffice for the cause, I guess we should use the `namespace` var here for Image-stream-namespace.